src/OrleansRuntime/GrainDirectory/LocalGrainDirectory.cs

```diff
@@ -1088,6 +1098,11 @@ internal IRemoteGrainDirectory GetDirectoryReference(SiloAddress silo)
             return InsideRuntimeClient.Current.InternalGrainFactory.GetSystemTarget<IRemoteGrainDirectory>(Constants.DirectoryServiceId, silo);
         }
 
+        private bool IsSiloNextInTheRing(SiloAddress siloAddr, int hash, bool excludeMySelf)
```

Comment:
I think that the resulting logic for this one-liner function will be more complex than now. If this is desired - I can change it.